src/coreComponents/physicsSolvers/contact/SolidMechanicsLagrangeContact.cpp

```diff
@@ -2262,7 +2262,8 @@ bool SolidMechanicsLagrangeContact::updateConfiguration( DomainPartition & domai
             integer const originalFractureState = fractureState[kfe];
             if( originalFractureState == FractureState::Open )
             {
-              if( dispJump[kfe][0] <= -normalDisplacementTolerance[kfe] )
+              // trying comparison in absolute
+              if( LvArray::math::abs( dispJump[kfe][0] ) <= LvArray::math::abs( normalDisplacementTolerance[kfe] ) )
```

Comment:
```suggestion
              if( LvArray::math::abs( dispJump[kfe][0] ) <= normalDisplacementTolerance[kfe]  )
```
I think the tolerance is always positive